tests/conftest.py

```diff
@@ -18,6 +18,9 @@
     get_real_invoice,
 )
 
+# dont install extensions for tests
+settings.lnbits_extensions_default_install = []
```

Comment:
- but this can be set in the `.env` file for tests, right?

Reply:
yes, but i think this is cleaner, because this actually is true for all tests